packages/web/src/components/Grid/README.md

```diff
@@ -0,0 +1,21 @@
+# Grid
```

Comment:
Please move `Grid` changes to a separate PR as discussed over Slack. It will make this PR smaller and easier to review.